ReplayKitExample/BroadcastExtensionSetupUI/BroadcastSetupViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BroadcastSetupViewController.swift
+//  BroadcastExtensionSetupUI
+//
+//  Copyright © 2018 Twilio. All rights reserved.
+//
+
+import ReplayKit
+
+class BroadcastSetupViewController: UIViewController {
+
+    @IBOutlet weak var broadcastButton: UIButton!
+    @IBOutlet weak var roomTextField: UITextField!
+
+    @IBAction func broadcast(_ sender: Any) {
+        userDidFinishSetup()
+    }
+
+    // Call this method when the user has finished interacting with the view controller and a broadcast stream can start
+    func userDidFinishSetup() {
+        // URL of the resource where broadcast can be viewed that will be returned to the application
+        let broadcastURL = URL(string:"https://s3-us-west-1.amazonaws.com/avplayervideo/What+Is+Cloud+Communications.mov")
```

Comment:
is the broadcastURL? Is there a way around it?

Reply:
I'm not sure about this one, at the time I was getting those invalid broadcast errors.